config/routes.rb

```diff
@@ -3,4 +3,5 @@
 
   post '/task', to: 'features#task_execution', as: :task_execution
   resources :features, except: :show
+  resources :site_notice
```

Comment:
may be that routes can use only for `index`.
example:
`resources :site_notice, only: :index`

Reply:
+1